README.md

```diff
@@ -109,3 +111,44 @@ docker-compose up
 ## Help and contact
 
 If you have any questions or suggestions, please [get in touch](https://ably.com/contact).
+# New Toolchain: Gatsby Documentation Repository
+
+This is a static site generated using [Gatsby](https://www.gatsbyjs.com/) and documentation written in:
+
+* [Textile](https://github.com/textile/textile-spec) - Held in the [./data/textile](./data/textile) folders, enhanced with certain additional features described in [./data/textile/nanoc-compatible/client-lib-development-guide/documentation-formatting-guide.textile](./data/textile/nanoc-compatible/client-lib-development-guide/documentation-formatting-guide.textile)
+
+## Current Status
+
+**Beta** - most patterns are now established, we are approaching v1.
+
+## Run
+
+`gatsby clean && gatsby develop`
```

Comment:
Think we need to mention node and npm are required? maybe we should add an asdf config so people don't have version problems too. And also, `npm i` :)

Reply:
Have added this, thanks